### PR TITLE
Add missing notifications for Location Services

### LINKED ISSUE #23: Check and prompt if the location is disabled
This should be helpful -
[https://stackoverflow.com/questions/41795968/react-native-check-and-prompt-user-to-enable-network-gps](https://stackoverflow.com/questions/41795968/react-native-check-and-prompt-user-to-enable-network-gps)

### PR DESCRIPTION
**Pull request by penrods**
Several conditions cause Private Kit to stop failing when it cannot access
GPS info.  We should notify the user when this has happened.  That includes:
* The app doesn't have proper permissions
* The permissions have been revoked
* The location service has been shut down

Still to do is a notification to show the user that the background service
is not doing anything when the user turns off location services.

 ### Testing Notes ###
Toggle permissions and location services, go in and out of the app.  There
should be notification within the app whenever it cannot access things.

 ### Fixed Issues ###
Resolves #23, #17

Leaving #50 open until we add a Notification that pops up

### CODE REVIEW

**Comment by tremblerz** [comment]
Testing it now. If I press No(on the application), leave the App and come back to the app, it doesn't ask to turn on locations now. Would say it's more of a feature creep for this one

File: app/services/LocationService.js [hunk 1]
```diff
@@ -64,21 +65,21 @@ export default class LocationServices {
             desiredAccuracy: BackgroundGeolocation.HIGH_ACCURACY,
             stationaryRadius: 50,
             distanceFilter: 50,
-            notificationTitle: 'PrivateKit Enabled',
-            notificationText: 'PrivateKit is recording path information on this device.',
+            notificationTitle: 'Private Kit Enabled',
+            notificationText: 'Private Kit is securely storing your GPS coordinates once every five minutes on this device.',
```
Comment (tremblerz) [inline app/services/LocationService.js]: Might not say securely storing as we are not encrypting the coordinates, they are stored in raw format. However, no other app can access it so it is good as well.

**Comment by penrods** [comment]
Yeah, I debated a little.  If you intentionally say "No", should we keep bugging you?  I think the Notifier 
will take care of this for us, too.

File: app/services/LocationService.js [hunk 2]
```diff
@@ -64,21 +65,21 @@ export default class LocationServices {
...
             debug: false,    // when true, it beeps every time a loc is read
             startOnBoot: false,
             stopOnTerminate: false,
             locationProvider: BackgroundGeolocation.ACTIVITY_PROVIDER,
 
             // DEBUG: Use these to get a faster output
-            //            interval: 2000,
-            //            fastestInterval: 2000, // Time (in milliseconds) between location information polls.  E.g. 60000*5 = 5 minutes
-            //            activitiesInterval: 2000,
+            interval: 2000,
```
Comment (tremblerz) [inline app/services/LocationService.js]: will comment it

**Comment by penrods** [comment]
Also, I got lazy around 5am.  ;-)

File: app/services/LocationService.js [hunk 3]
```diff
@@ -195,10 +203,39 @@ export default class LocationServices {
...
+                        onPress: () => BackgroundGeolocation.showLocationSettings()
+                    },
+                    {
+                        text: 'No',
+                        onPress: () => console.log('No Pressed'),
+                        style: 'cancel'
+                    }
+                    ]), 1000);
+            }
+            else if (!status.authorization) {
```
Comment (tremblerz) [inline app/services/LocationService.js]: If I don't authorize and press Yes for settings, then I end up at App settings which doesn't have location information anywhere.

**Pull request merged**